### DISCUSSION TITLE
Siemens S7 read data problem

### DISCUSSION THREAD

**Discussion by guang067** [frangoteam/FUXA#1031] [Q&A]
Dear all.
Ask for help.  i have a Siemens  PLC and got a DB as below:
![image](https://github.com/frangoteam/FUXA/assets/6702824/fe05cb4b-a0dd-4e6c-abcb-49e31d40a3d1)

i have connect plc via fuxa success, 
![image](https://github.com/frangoteam/FUXA/assets/6702824/33fbccb1-6164-43c9-b8ff-b340a76691cb)

seems by add tag with DB3.Done can not get the value ,  then how can i read data from it ? 
![image](https://github.com/frangoteam/FUXA/assets/6702824/2bd48480-92bb-49bd-944b-2879317c117b)

**Comment by unocelli** [comment]
Hi, it doesn't seem to me that db3.Done is an address of the S7 standard.
